### PR TITLE
Add /api/get-club servlet which returns the JSON representation of a club given that club's ID.

### LINKED ISSUE #57: Create process for getting a Club object through web API
/get-club should retrieve a Club object that is constructed from information in the database Clubs table using the `fetchClubFromId` method defined in the Storage Handler API.

### PR DESCRIPTION
**Pull request by JosephBushagour**
This servlet will be used when retrieving specific information about one club on the frontend.

### CODE REVIEW

File: server/src/main/java/com/google/coffeehouse/servlets/GetClubServlet.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  /** 
+   * Returns a {@link Club} object in JSON format from information in the database.
+   * @param request the GET request that must have a valid JSON representation of the "clubId" to
+   *     be passed in order to fetch a Club from ID in the database. If this is not the case the
+   *     response will send a "400 Bad Request error". If an error occurs when attempting to fetch
+   *     the Club from the database, the response will send a "500 Internal Server error" 
+   * @param response the response from this method, will contain the object in JSON format.
+   *     If the request object does not have a valid JSON body that describes the Club, this object
+   *     will send a "400 Bad Request error" If an error occurs when attempting to fetch
+   *     the Club from the database, this object will send a "500 Internal Server error"
```
Comment (linamontes10-zz) [inline server/src/main/java/com/google/coffeehouse/servlets/GetClubServlet.java]: Make sure to add the SC_NOT_FOUND error which is encountered if the club does not exist in the database. Something along the lines of: "If the request object is attempting to get a club which does not exist in the database, the response will send a "404 Not Found error".

File: server/src/test/java/com/google/coffeehouse/servlets/GetClubServletTest.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    assertEquals(OWNER_ID, c.getOwnerId());
+    assertEquals(CLUB_ID, c.getClubId());
+    assertEquals(DESCRIPTION, c.getDescription());
+    assertEquals(testContentWarnings, c.getContentWarnings());
+    assertEquals(BOOK_TITLE, c.getCurrentBook().getTitle());
+    assertEquals(BOOK_ID, c.getCurrentBook().getBookId());
+  }
+
+  @Test
+  public void doGet_noUserId() throws IOException {
```
Comment (linamontes10-zz) [inline server/src/test/java/com/google/coffeehouse/servlets/GetClubServletTest.java]:
```suggestion
  public void doGet_noClubId() throws IOException {
```

**Review by linamontes10-zz — APPROVED** [review]
Looks great :)

File: server/src/main/java/com/google/coffeehouse/servlets/GetClubServlet.java (hunk 1 above)
Comment (JosephBushagour) [reply server/src/main/java/com/google/coffeehouse/servlets/GetClubServlet.java]: Good catch! This has been added and pushed.

File: server/src/test/java/com/google/coffeehouse/servlets/GetClubServletTest.java (hunk 2 above)
Comment (JosephBushagour) [reply server/src/test/java/com/google/coffeehouse/servlets/GetClubServletTest.java]: Thanks! Fix pushed.

**Review by jimmycolema — APPROVED** [review]
lgtm

**Review by KeatonTech — APPROVED** [review]

**Review by SarahGB — APPROVED** [review]

**Pull request merged**